advanced-multi-block.php

```diff
@@ -13,25 +13,29 @@
  * @package CreateBlock
  */
 
-if (! defined('ABSPATH') ) {
-  exit;
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
 }
 
-// Include Composer's autoload file.
-if ( file_exists( plugin_dir_path( __FILE__ ) . 'vendor/autoload.php' ) ) {
-  require_once plugin_dir_path( __FILE__ ) . 'vendor/autoload.php';
-} else {
-  wp_trigger_error( 'Advanced Multi Block Plugin: Composer autoload file not found. Please run `composer install`.', E_USER_ERROR );
-  return;
+// Include our bundled autoload if not loaded globally.
+if ( ! class_exists( Advanced_Multi_Block\Plugin_Paths::class ) && file_exists( __DIR__ . '/vendor/autoload.php' ) ) {
```

Comment:
Here we attempt to require the bundled `vendor/autoload.php` _only_ if the class is not resolved by the global project autoloader. By default, calling `class_exists()` triggers the autoload logic so that attempts everything that a global Composer autoload has registered (if at all).